### PR TITLE
WIP: Fiona edits

### PR DESCRIPTION
**Pull request by fionahanington**
fix typo; add make file for windows

### CODE REVIEW

**Comment by fionahanington** [comment]
Alex pls assign reviewer

File: dimod/template_sampler.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def sample_qubo(self, Q, **kwargs):
                 sample_ising.
 
         Returns:
-            :obj:`BinaryResponse`:
+            :obj:`.BinaryResponse`:
```
Comment (orenshk) [inline dimod/template_sampler.py]: Why does this require a leading dot?

Comment (arcondello) [reply dimod/template_sampler.py]: Because in order for sphinx to make the link it needs to know to look in `dimod.BinaryResponse`

**Comment by fionahanington** [comment]
I did this change with Alex - the hotlink wasn't working otherwise

File: dimod/template_sampler.py (hunk 1 above)
Comment (fionahanington) [reply dimod/template_sampler.py]: I did this change with Alex - the hotlink wasn't working otherwise

Comment (orenshk) [reply dimod/template_sampler.py]: Oh so `.BinaryResponse` is short hand for `dimod.BinaryResponse`? Neat-o

Comment (arcondello) [reply dimod/template_sampler.py]: I told it where to look in the .rst file `.. currentmodule:: dimod`. Or at least I am pretty sure that's how that works...

**Pull request closed without merging**